### PR TITLE
Add database seed with all question types form

### PR DESCRIPTION
**Pull request by lfdebrux**
### What problem does this pull request solve?

Make it quicker to get a useful test form created when doing local development.

This PR adds a copy of @DavidBiddle's 'All question types' form that already exists in dev at `/forms/71/live`. If you want to add this to your database you can simply run `rails db:seed`

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

**Review by chao-xian — DISMISSED** [review]
TYSM for doing this!!!! Love it!

**Review by DavidBiddle — APPROVED** [review]
This looks good, and a very nice feature to have - only thing worth noting is that it doesn't include a selection question, so 'all question types form' is a bit of a misnomer (which is a fault with the original form rather than with this PR)

**Comment by lfdebrux** [comment]
> This looks good, and a very nice feature to have - only thing worth noting is that it doesn't include a selection question, so 'all question types form' is a bit of a misnomer (which is a fault with the original form rather than with this PR)

Oh good catch. I'll add a selection question now.

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,83 @@
...
+    ),
+    Page.create(
+      question_text: "Phone number",
+      answer_type: "phone_number",
+    ),
+    Page.create(
+      question_text: "Selection from a list of options",
+      answer_type: "selection",
+      answer_settings: {
+        "only_one_option": "0", # TODO: investigate why we set this to "0"
```
Comment (lfdebrux) [inline db/seeds.rb:59]: I've created a ticket for this: https://trello.com/c/prWyiTVd/1310-forms-admin-saves-onlyoneoption-0-into-database-if-user-does-not-check-box

**Comment by lfdebrux** [comment]
@DavidBiddle I've added a selection question, while doing so I found a curiosity which I have recorded on Trello. Would you mind reviewing, or can I get a re-review from @chao-xian?

File: db/seeds.rb (hunk 1 above)
Comment (DavidBiddle) [reply db/seeds.rb:59]: I think I probably introduced this - I vaguely remember running into trouble with these boolean fields and ActiveResource. There's almost certainly a better way of doing it, so I appreciate you adding the card to investigate it!

**Review by DavidBiddle — APPROVED** [review]

**Pull request merged**